src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+import { Post } from '../../types/Post';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      {' Posted by  '}
+
+      <UserInfo user={post.user} />
```

Comment:
Should we render the user component if we have no user?

```suggestion
      {post.user && (<UserInfo user={post.user} />) }
```

Reply:
my bad, same issue like last task